tests/io/test.directory.js

```diff
@@ -198,3 +201,19 @@ describe('Directory.getFileAsString()', function() {
   });
 
 });
+
+
+describe('Directory.getChunkAsBuffer()', function() {
+
+  it('should get a buffer', () => {
+    var myDirectory = new Directory('tests/fixtures/io/');
+    return myDirectory.getFiles()
+      .then(() => {
+        return myDirectory.getChunkAsBuffer('dir2/dir3/file3.txt', 2);
```

Comment:
I suppose chunks could be part of an options object then it would be a more obvious signature.